pkg/cli/host_preflight_test.go

```diff
@@ -45,7 +45,7 @@ func TestNewHostPreflightCmd(t *testing.T) {
 					IsPass:  true,
 				},
 			},
-			stdout: green("[PASS]") + " Number of CPUs: At least 4 CPU cores are required\n",
+			stdout: OutputPassGreen() + " Number of CPUs: At least 4 CPU cores are required\n",
```

Comment:
I would GREATLY prefer for this not to call `OutputPassGreen` etc

Reply:
A() = A() isn't a good unit test IMO

Reply:
That is how that was before. I think has no point to change the tests now.